### PR TITLE
Refactor : combine the duplicated license check plugin

### PR DESCRIPTION
**Pull request by taojintianxia**
Changes proposed in this pull request:
  - upgrade skywalking-eyes to 0.4.0
  - combine the skywalking-eyes and Apache RAT plugin

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have passed maven check: `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: .github/workflows/checkstyle-and-license-check.yml [hunk 1]
```diff
@@ -18,15 +18,15 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: Check:Checkstyle and Apache Rat Check
+name: Check:Checkstyle and License Check
```
Comment (terrymanu) [inline .github/workflows/checkstyle-and-license-check.yml:21]: How about rename as `Checkstyle and License Check`

Comment (taojintianxia) [reply .github/workflows/checkstyle-and-license-check.yml:21]: the `Check:` is a prefix, if there are more than 2 checks action, it's easy to group these action together. like followings

image

Comment (terrymanu) [reply .github/workflows/checkstyle-and-license-check.yml:21]: OK, then, how about rename as `Check: Checkstyle and Licenses`

Comment (terrymanu) [reply .github/workflows/checkstyle-and-license-check.yml:21]: By the way, ":" should keep blank with follow chars. Please change `Docker: xxx` as well.

Comment (taojintianxia) [reply .github/workflows/checkstyle-and-license-check.yml:21]: it's impossible . casue in yaml file `:` with blank space will not be treated as a variable.
image
image

**Comment by taojintianxia** [comment]
image

**Review by terrymanu — APPROVED** [review]

**Pull request merged**